web/pages/Chub/CharList.tsx

```diff
@@ -0,0 +1,39 @@
+import { Component, For, Show, createEffect, createMemo } from 'solid-js'
+import { ChubItem } from './ChubItem'
+import { chubStore } from '../../store/chub'
+import ChubNavigation, { chubPage } from './ChubNavigation'
+
+const CharList: Component = () => {
+  createEffect(() => {
+    chubStore.getChubChars()
+  })
+
+  const chars = createMemo(() => {
+    return chubStore().chars
+  })
+
+  return (
+    <>
+      <ChubNavigation buttons={chars()!.length >= 48} />
+      <div class="grid w-full grid-cols-[repeat(auto-fit,minmax(105px,1fr))] flex-row flex-wrap justify-start gap-2 py-2">
+        <For each={chars().slice(48 * (chubPage() - 1))}>
+          {(char) => (
+            <ChubItem
+              name={char.name}
+              fullPath={char.fullPath}
+              avatar={
+                `https://avatars.charhub.io/avatars/${char.fullPath}/avatar.webp` ||
+                `https://git.characterhub.org/${char.fullPath}/-/raw/main/avatar.webp`
+              }
+            />
+          )}
+        </For>
+        <Show when={chars.length < 4}>
```

Comment:
Same here with `chars().length` if you're referencing a memoized value. If you change `chars` as suggested above then this will be correct.